.travis.sh

```diff
@@ -86,6 +86,8 @@ case "$JDK" in
   ;;
 9-ea | 9-ea-stable)
   # see https://bugs.openjdk.java.net/browse/JDK-8131041 about "java.locale.providers"
+  # Groovy version should be updated to get rid of "--add-opens" options (see https://twitter.com/CedricChampeau/status/807285853580103684)
+  MAVEN_OPTS="--add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.io=ALL-UNNAMED" \
   mvn -V -B -e verify -Dbytecode.version=1.9 \
```

Comment:
Very minor, but perhaps indent the `mvn` line?  The trailing `\` on the `MAVEN_OPTS` line can be difficult to spot.